Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2588 @@
+        cmd_args=args,
+        readable_title='Domain Instances',
+        context_path='DomainInstances',
+        output_key_field='data_source_url_domain',
+        command_type='incident',
+        is_call_diff_readable_output=True,
+        func_readable_output=incident_comment_readable_output,
+        uuid=uuid,
+        incident_id=incident_id)
+
+
+def patch_incident_update_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+      Incident Update command is used to Add, close or update incident resolution
+      Args:
+          client: client object to use.
+          args: all command arguments, usually passed from ``demisto.args()``.
+      Returns:
+          CommandResults: A ``CommandResults`` object that is then passed to ``return_results``,
+          that contains an updated
+              result.
+    """
+    action = args.get('operation')
+    update_value = args.get('value', '')
+
+    if action not in INCIDENT_PATCH_ACTION:
+        raise ValueError(f'Invalid Incident Patch Operation: Supported values are : {INCIDENT_PATCH_ACTION}')
+
+    # Get UUID based on incident_id
+    if not (uuid := get_incident_uuid(client, args)):
+        raise ValueError(f'Incident ID {args.get("incident_id")} was not found. '
+                         f'If it\'s older than 30 days, try increasing the time range arguments')
+    # Incident Add Comment
+    if action == 'add_comment':
+        if not update_value:
+            raise ValueError('Missing comment. Enter Free text up to 512 characters')
+
+        action_desc = 'Add Comment'
+        response = client.add_incident_comment(uuid, update_value)
+        status = response.status_code
+
+        # Incident Close Incident
+    elif action == 'close_incident':
+        action_desc = 'Close Incident'
+        response = client.close_incident(uuid, 4)
+        status = response.status_code
+
+        # Incident Update Resolution
+    elif action == 'update_resolution':
+        action_desc = 'Update Status'
+        if not update_value or INCIDENT_RESOLUTION.get(str(update_value)) is None:
+            raise ValueError(f'Missing/Invalid Incident Resolution value. '
+                             f'Enter any one of these supported value {INCIDENT_RESOLUTION}')
+        response = client.update_incident(uuid, int(args.get('value', 0)))
+        status = response.status_code
+    else:
+        raise DemistoException(
+            f'Unable to perform Incident update. Only support the following action {INCIDENT_PATCH_ACTION}')
+
+    if status == 204:
+        summary_data = {
+            'incident_id': args.get('incident_id'),
+            'Message': 'Successfully Updated',
+        }
+        headers = list(summary_data.keys())
+        readable_output = tableToMarkdown(f'Incident {action_desc}', summary_data, headers=headers, removeNull=True)
+    else:
+        readable_output = f'Failed {action}. Response from endpoint {status}'
```

Comment:
`status` is a number - is this the design?

Reply:
Resolved, kind review.